src/fides/data/sample_project/cookie_house/src/components/Home/index.tsx

```diff
@@ -11,61 +11,67 @@ interface Props {
 }
 
 const Home = ({ products }: Props) => {
-  const [productInPurchase, setProductInPurchase] = useState<Product | null>(null);
+  const [productInPurchase, setProductInPurchase] = useState<Product | null>(
+    null,
+  );
   const [popupTrigger, setPopupTrigger] = useState(0);
   const onCloseModal = useCallback(() => {
     setProductInPurchase(null);
   }, []);
 
   const onSubmit = useCallback(async (data: UserData) => {
-    await (await fetch('/api/purchase', {
-      method: 'POST',
-      body: JSON.stringify(data),
-      headers: new Headers({
-        'Content-Type': 'application/json',
-        Accept: 'application/json',
-      }),
-    })).json();
+    await (
+      await fetch('/api/purchase', {
+        method: 'POST',
+        body: JSON.stringify(data),
+        headers: new Headers({
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+        }),
+      })
+    ).json();
 
-    setPopupTrigger(v => v + 1);
+    setPopupTrigger((v) => v + 1);
 
     setProductInPurchase(null);
   }, []);
 
   return (
     <div className={css.main}>
-        <Header />
-        <main>
-            <div className={css.products}>
-            {products.map((product) => (
-                <div key={product.id} className={css.product}>
-                    {/* eslint-disable-next-line @next/next/no-img-element */}
-                    <img src={product.url} alt={product.name ?? 'Cookie'} />
-                    <div className={css.name}>{product.name}</div>
-                    <div className={css.description}>{product.description}</div>
-                    <div className={css.price}>{product.price}</div>
-                    <Button color="primary" 
-                      className={css.purchase} 
-                      onClick={() => setProductInPurchase(product)}
-                    >
-                      Purchase
-                    </Button>
-                </div>
-            ))}
+      <script src="http://localhost:3001/fides-consent.js" async></script>
```

Comment:
this line is my only addition the rest are prettier changes as the cookie house app has different setting